iommi/admin.py

```diff
@@ -409,10 +410,10 @@ class LoginForm(Form):
     password = Field.password()
 
     class Meta:
-        title = 'Login'
+        title = _('Login')
 
         @staticmethod
-        def actions__submit__post_handler(form, **_):
+        def actions__submit__post_handler(form, **__):
```

Comment:
No?

Reply:
Ah, ok I see... But should we this everywhere? Is this the new black?

Reply:
You're right, it's a bad idea. Explicit gettext is better.